layouts/New/layouts.module.css

```diff
@@ -216,3 +216,52 @@
     }
   }
 }
+
+.notFoundLayout {
+  @apply flex
+    items-center
+    justify-center
+    xs:px-4
+    xs:py-8;
+
+  main {
+    @apply flex
+      max-w-xl
+      flex-col
+      items-center
+      gap-0
+      text-center;
+
+    h1 {
+      @apply bg-gradient-to-b
```

Comment:
There's a class name on effects.css that is used for this h1 with effect :) no need to reapply the styles here